Update README.md

shestee opened the pull request:
Updated documentation on recently added "no version property" feature that allows deducing version based on annotated tag.

shestee committed:
Update README.md

Added documentation on "no version" feature.

mockitoguy commented on README.md:
```diff
@@ -19,9 +19,9 @@ Encourage and help software developers set up their releases to be fully automat
 # shipkit-auto-version Gradle Plugin
```
Can you also update the "shipkit-auto-version Gradle Plugin" section? That section indicates that you *need* a `version.properties` file. Perhaps we can simplify this section to avoid duplication with "Usage" section.

mockitoguy commented on README.md:
```diff
@@ -38,8 +38,13 @@ Check out [shipkit-changelog](https://github.com/shipkit/shipkit-changelog) plug
+- https://github.com/mockito/mockito-kotlin
 
-## Usage
+## USAGE
```
Nit: Let's use lowercase consistently (or uppercase), e.g. `Usage` here or `CAPITAL` everywhere else for headers

shestee replied:
Thanks for review! The documentation is now updated with your suggestions.

mockitoguy reviewed (approved):
Nice! Thanks!

shestee committed:
Update README.md general section

Simplified 'about' section of README.md and added short info about new feature. Details are contained in 'usage' section.
